internal/server/routes.go

```diff
@@ -251,17 +225,15 @@ func redirectsFor(a *API, method, path string) []apiMigration {
 		if strings.ToUpper(migration.method) != method {
 			continue
 		}
-		if migration.redirect != path {
-			continue
-		}
-		if len(migration.redirect) > 0 {
+		// TODO: also check version
+		if migration.rename == path {
```

Comment:
A small simplification here. We can check redirect once for equality. `path` has to be a non-empty string, so this will never compare to non-rename migrations.